<launch>

<node name="driver"      pkg="face_detection" type="faceDetectionDriver" output="screen" />
<node name="application" pkg="face_detection" type="faceDetection"       output="screen" />
<node name="stub"        pkg="face_detection" type="faceDetectionStub"   output="screen" />


</launch>